hack/local-up-karmada.sh

```diff
@@ -74,7 +74,11 @@ ${KARMADACTL_BIN} join --karmada-context="${KARMADA_APISERVER_CLUSTER_NAME}" ${M
 "${REPO_ROOT}"/hack/deploy-scheduler-estimator.sh "${MAIN_KUBECONFIG}" "${HOST_CLUSTER_NAME}" "${MEMBER_CLUSTER_KUBECONFIG}" "${MEMBER_CLUSTER_2_NAME}"
 
 # step4. register pull mode member clusters and install scheduler-estimator
-"${REPO_ROOT}"/hack/deploy-agent-and-estimator.sh "${MAIN_KUBECONFIG}" "${HOST_CLUSTER_NAME}" "${MAIN_KUBECONFIG}" "${KARMADA_APISERVER_CLUSTER_NAME}" "${MEMBER_CLUSTER_KUBECONFIG}" "${PULL_MODE_CLUSTER_NAME}"
```

Comment:
Overall LGTM, but this seems to mean that we will lose control of `/hack/ deploy-agent-and-estimater.sh`, i.e. the script may become obsolete without anyone knowing because it is not used in CI.

or we have any plan to remove `/hack/ deploy-agent-and-estimater.sh`?

Reply:
Hi, long time no see 😄 
nice catch, I'll add this in #5916